### PR TITLE
ISSUE-20296: "@magentoDataIsolation" is used instead of "@magentoDbIsolation" in some integration tests.

### PR DESCRIPTION
**Pull request by p-bystritsky**
### Fixed Issues (if relevant)

1. magento/magento2#20296: "@magentoDataIsolation" is used instead of "@magentoDbIsolation" in some integration tests.

### Manual testing scenarios (*)

No scenario

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @p-bystritsky. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.2-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: dev/tests/integration/testsuite/Magento/CatalogImportExport/Model/Import/ProductTest.php [hunk 1]
```diff
@@ -741,7 +741,7 @@ protected function getOptionValues(\Magento\Catalog\Model\Product\Option $option
     /**
      * Test that product import with images works properly
      *
-     * @magentoDataIsolation enabled
+     * @magentoDbIsolation enabled
```
Comment (orlangur) [inline dev/tests/integration/testsuite/Magento/CatalogImportExport/Model/Import/ProductTest.php]: So, it didn't have effect before? Why do we need to enable DB isolation then?

**Review by orlangur — CHANGES_REQUESTED** [review]

File: dev/tests/integration/testsuite/Magento/CatalogImportExport/Model/Import/ProductTest.php (hunk 1 above)
Comment (p-bystritsky) [reply dev/tests/integration/testsuite/Magento/CatalogImportExport/Model/Import/ProductTest.php]: @orlangur updated.

**Review by orlangur — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @orlangur, thank you for the review.
[ENGCOM-3875](https://jira.corp.magento.com/browse/ENGCOM-3875) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @p-bystritsky. Thank you for your contribution. 
We will aim to release these changes as part of 2.2.8. 
Please check the release notes for final confirmation.

Please, consider to port this solution to 2.3 release line.
You may use [Porting tool](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#porting-tool) to port commits automatically.

**Comment by p-bystritsky** [comment]
Port to 2.3 is not needed as issue is already fixed in 2.3-develop branch (commit ab4ad4d63a4724d03f52f6a42510c0dacbb83e4f).

**Pull request merged**